run-tests.sh

```diff
@@ -1,16 +1,23 @@
-#!/bin/sh
+#!/bin/bash
 #
 # This file is part of REANA.
 # Copyright (C) 2017, 2018 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+
+cli_docs_url=https://raw.githubusercontent.com/reanahub/docs.reana.io/master/docs/reference/reana-client-cli-api/index.md
+docs_differ_error_msg='Current reana-client differs with the documentation. Please update http://docs.reana.io/reference/reana-client-cli-api/.'
+
 pydocstyle reana_client && \
 isort -rc -c -df **/*.py && \
 reana-client --help > cmd_list.txt && \
 diff -q -w docs/cmd_list.txt cmd_list.txt  && \
 rm cmd_list.txt && \
+python scripts/generate_cli_api.py > cli_api.md && \
+(diff -q -w  cli_api.md <(curl -s $cli_docs_url) || (echo $docs_differ_error_msg && exit 1)) && \
+rm cli_api.md && \
```

Comment:
Put `cli_api.md` into `.gitignore` in case something fails before it is removed?

Alternatively use `/tmp/cli_api.md` place for this "temporary" file?